packages/js/src/analysis/collectAnalysisData.js

```diff
@@ -72,6 +72,7 @@ export default function collectAnalysisData( editorData, store, customAnalysisDa
 	// Make a data structure for the paper data.
 	const data = {
 		text: editData.content,
+		textTitle: editData.title,
```

Comment:
Just curious: any reason why not just use `title` as attribute name?

Reply:
Because we saved it as `title` in the editor data store 🙈. For context, it was already named like that, not in this PR. I know it's confusing in general that we have title (which is SEO title) and text title, but sometimes we use the same key name for both. But I think if we want to improve this, we should do it in a separate PR.